### ISSUE TITLE
Perfect rewriting isn't perfect

### ISSUE THREAD

**Issue by ricroberts** [Swirrl/drafter#571]
If you have a values clause in a query, and we pass in a values clause e.g. in [this query](https://github.com/Swirrl/muttnik/blob/95f0a8059f312f29fc2387af4cc02a4a54100fb7/src/mut/admin/lookup-catalog-entries-for-graphs.sparql) ...then any live graphs passed in (which have changed) will be rewritten into the draft-graphs. All ok so far.

But if the metadata graph hasn't itself changed then there wont be a draft graph for it, with a pre-rewritten version of the ?`ds_graph` in the db for this clause to match:

`?catalog_entry pmdcat:graph ?ds_graph .`

So basically rewriting isn't actually 'perfect' when spanning across live and draft graphs, if you're trying to match graph identifiers.
